### PR TITLE
Fix getting started pull request

### PR DESCRIPTION
**Pull request by aminalhazwani**
Changelog: 
- Remove inline style, columns, emoji, etc. I would focus on the contents now. We can agree on style at a later point, maybe during work week
- Remove comments
- Remove `<br>` and non-breaking spaces
- Replace section `h1`s with `h2`s in order to respect heading hierarchy
- Add a temporary button
- Fix minor style issues in the page header and section

cc @designakt

### CODE REVIEW

**Comment by designakt** [comment]
I don't understand why we would need to revert that?
Those pages do not need a TOC and are readable as they are. 
Is anything causing a problem for the DS?

File: contents/getting-started.html [hunk 1]
```diff
@@ -1,46 +1,46 @@
...
+  <p>
     The Firefox Design System provides a centralized, living source for look, feel and sound of our products  to create consistent user interfaces across all Firefox products.<br/>
     We collect reusable UI components, templates and other resources to help you focus on the unique parts of your work.
   </p>
 </header>
 
-<section style="column-count: 2; column-gap: 2em;">
-    <h1>🐺 for Developers</h1>
+<section>
+    <h2>For Developers</h2>
```
Comment (bwinton) [inline contents/getting-started.html]: Boo.  I liked the emoji.  😉

File: contents/welcome.html [hunk 2]
```diff
@@ -1,54 +1,44 @@
...
 
-<section style="text-align: center;">
-    <h1 style="margin-top:2rem; margin-bottom:4rem; font-size:1.5rem;">
-      <a href="/getting-started.html" style="background-color: var(--blue-5); padding: 1rem 2rem;  color: white;">
-        Get started!
-      </a>
-    </h1>
+<section>
+  <button>
+    <a href="/getting-started.html">Get started</a>
```
Comment (bwinton) [inline contents/welcome.html]: Should just be `<a href="getting-started.html">Get started</a>`.

Comment (aminalhazwani) [inline contents/welcome.html]: I tried but still it doesn't go to the desired page, no idea 🤔

File: contents/getting-started.html (hunk 1 above)
Comment (aminalhazwani) [inline contents/getting-started.html]: Everybody loves emoji, and I do too 😺 I just think that at the moment we should merely focus on the contents. And later talk all together about style, layouts, emoji, typography, etc.

**Comment by aminalhazwani** [comment]
@designakt regarding the table of contents I would prefer to leave things as they are. Following our request Erica invested a great effort in automatizing its generation. I would avoid hacky workarounds in order to hide it. I think we can postpone this decision, when we are gonna have a greater amount of content we could fully evaluate if it's necessary to hide the toc from certain pages

File: contents/getting-started.html (hunk 1 above)
Comment (designakt) [inline contents/getting-started.html]: Why do you think we should only focus on content?
And aren't emoji content? 😉

**Comment by designakt** [comment]
> regarding the table of contents I would prefer to leave things as they are.
Why would you prefer that? The TOC does not really add to those pages, but makes them more difficult to consume.
How about we leave it as it is, and as soon as we have a proper technical solution to display this as it is now, we change the code? (I can fix the broken link and the mail address separately.)

**Comment by aminalhazwani** [comment]
@designakt what you are proposing sounds more like graceful degradation rather than progressive enhancement. We are basically adding changes that we already know we will have to remove at a later point. And overwriting features that we specifically requested. 

Personally I do not see any impediment in consuming the content with the TOC visible. Actually it gives a great overview of the contents below the fold, especially on small screens. 

Broken link has already been fixed in this pull request 😉

**Comment by aminalhazwani** [comment]
@designakt add responsive two/three-col layout and emoji as placeholders 😉 Will merge it!

**Pull request merged**